docker/Dockerfile.devel

```diff
@@ -89,4 +89,6 @@ RUN set -ex && mkdir build_in_ci && cd build_in_ci &&
       .. &&                                                                           \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install
 
+ENV LD_LIBRARY_PATH=/opt/tritonserver/lib:/opt/tritonserver/backends/dali/dali:${LD_LIBRARY_PATH}
```

Comment:
Why it is needed?

Reply:
It's needed to run unittests binary. It dynamically links Triton and DALI libs which are not in LD_LIBRARY_PATH by default